aws/README.md

```diff
@@ -0,0 +1,54 @@
+# AWS CLI v2
```

Comment:
This seems very environment specific. For example, `rm /usr/local/bin/aws` ... will the binary always be installed there?